client/state/happychat/selectors/test/get-geolocation.js

```diff
@@ -0,0 +1,35 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import getGeoLocation from '../get-geolocation';
+
+describe( 'getGeoLocation', () => {
+	test( 'should return null if geoLocation is not set', () => {
+		const selected = getGeoLocation( {
+			happychat: {
+				user: { geoLocation: null },
+			},
+		} );
+		expect( selected ).to.equal( null );
+	} );
+
+	test( 'should return value if geoLocation is set', () => {
+		const selected = getGeoLocation( {
+			happychat: {
+				user: {
+					geoLocation: {
+						city: 'Timisoara',
+					},
+				},
+			},
+		} );
+		expect( selected.city ).to.equal( 'Timisoara' );
```

Comment:
This might look more accurate if we did `expect( selected ).to.eql( { city: 'Timisoara' } );`